development/tools/jobs/kyma/kyma_integration_test.go

```diff
@@ -263,7 +263,7 @@ func TestKymaIntegrationJobPeriodics(t *testing.T) {
 	require.NotNil(t, vmsCleanerPeriodic)
 	assert.Equal(t, expName, vmsCleanerPeriodic.Name)
 	assert.True(t, vmsCleanerPeriodic.Decorate)
-	assert.Equal(t, "30 */4 * * 1-5", vmsCleanerPeriodic.Cron)
+	assert.Equal(t, "0 */1 * * *", vmsCleanerPeriodic.Cron)
```

Comment:
*/1  == *

Reply:
Sure, IMO `*/1` is more readable. I changed it anyway.